### PR TITLE
Fix broken links in SDK docs v2.3.8 - Closes #992 

### LINKED ISSUE #992: Fix broken links in SDK docs v2.3.8
Fix the following broken links:

v2/index.html
v2/reference/api.html
v2/reference/lisk-framework/index.html
v2/setup.html
v2/tutorials/index.html

### PR DESCRIPTION
**Pull request by chris529**
Closes #992

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -5,9 +5,6 @@ Mona Bärenfänger <mona@lightcurve.io>
 :imagesdir: ../assets/images
 :v_core: master
 :experimental:
-:page-no-previous: true
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc:8]: this line should stay

File: modules/ROOT/pages/reference/lisk-framework/index.adoc [hunk 2]
```diff
@@ -4,9 +4,6 @@ Mona Bärenfänger <mona@lightcurve.io>
 :toc:
 :imagesdir: ../../assets/images
 :v_core: master
-:page-no-next: true
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-framework/index.adoc:7]: this line should stay

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc:8]: Ah yes, now done 👍

File: modules/ROOT/pages/reference/lisk-framework/index.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-framework/index.adoc:7]: Also now done 👍

**Review by Tschakki — APPROVED** [review]

**Pull request merged**